content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -87,7 +87,8 @@ For example:
     },
     {
       "type": "portmap",
-      "capabilities": {"portMappings": true}
+      "capabilities": {"portMappings": true},
+      "externalSetMarkChain": "KUBE-MARK-MASQ"
```

Comment:
I am not sure if this is necessary, hoping someone from @kubernetes/sig-network-pr-reviews can take a look